app/src/main/java/com/example/weather/network/Dao.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.weather.network
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+
+@Dao
+interface CityDao {
+
+    @Query("SELECT * FROM city_table ORDER BY name ASC")
+    fun getCities(): List<CachedCityOverview>
+
+    @Insert(onConflict = OnConflictStrategy.IGNORE)
```

Comment:
Do we want to ignore the new data if we already have old data in the db? Or do we want to always update the data in the database so it has the most up to date weather data